docs/api/connections.rst

```diff
@@ -403,6 +403,31 @@ The `!Connection` class
         .. |pg_prepared_xacts| replace:: `pg_prepared_xacts`
         .. _pg_prepared_xacts: https://www.postgresql.org/docs/current/static/view-pg-prepared-xacts.html
 
+    .. _raw-queries:
+
+    Raw Queries
+    -----------
+
+    .. versionadded:: 3.1
+
+    Setting `raw_queries` to `!True` during connection allows users to use
```

Comment:
I think that raw_query should be at most a cursor property, not so much a connection property. The cursors are in charge of making queries and return results, the connections are in charge of keeping connection and transaction state.

We don't have concepts such as server-side vs. client-side querying, server- vs. client- side binding, binary parameters etc. in the connection. So I think raw_query should be at most at `conn.cursor()` level, not higher than that.

Reply:
Thanks for your feedback! I understand your perspective on classifying raw_query as a cursor property. However, I think raw_query has a relational context with both cursors and connections - you might want to enable it for all or just one cursor in a connection, so I think from a relational point of view, it makes sense to talk about "raw_query" for a connection.

My main need is a simple way to enable raw_query for all queries in a connection without altering all cursor() or execute() calls.

The changes to _queries.py shouldn't be affected by the interface design, perhaps that could be reviewed first, to see if it's a viable approach.

Reply:
The use case of having all the cursors the same way should be covered by the cursor_factory property.